mrf/era-1/basic-syntax-#9.js

```diff
@@ -0,0 +1,131 @@
+// add number a to b
+function add(a, b) {
+  return a + b;
+}
+
+// flip a coin and log the result (head or tail)
+function flipACoin() {
+  let result = Math.random();
+
+  if (result >= 0.5) return "HEAD";
+  else return "TAIL";
+}
+
+// concat two strings
+function concat(stringA, stringB) {
+  // return stringA + stringB;
+  return stringA.concat(stringB);
+}
+
+// 2^x = ?
+function power2(x) {
+  // return Math.pow(2, x);
+  return 2 ** x;
+}
+
+// user object with first name and last name, both capitalized
+// e.g. getUser('hello', 'world') is { firstName: 'Hello', lastName: 'World' }
+function getUser(firstName, lastName) {
+  firstName = firstName.charAt(0).toUpperCase() + firstName.slice(1);
+  lastName = lastName.charAt(0).toUpperCase() + lastName.slice(1);
+  const arr = { firstName: firstName, lastName: lastName };
+  return arr;
+}
+
+// greet someone!
+// e.g. greet('ConnecMent') is "Hello, ConnecMent! Hope you the best."
+function greet(name) {
+  // return "Hello, " + name + " Hope you the best.";
+  return `Hello, ${name} Hope you the best.`;
+}
+
+// get array of n random numbers
+function getNRandomNumbers(n) {
+  const final = new Array(n).fill();
+
+  // for (let i = 0; i < n; i++) {
+  //   final[i] = Math.random();
+  // }
+  // return final;
+
+  // or
+  return final.map(() => Math.random());
```

Comment:
Choose only one of the return statements. Do not apply suggestions blindly.

Reply:
ok